CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+Version 0.3.1 *(08/02/2017)*
```

Comment:
There is no automatic trigger at the moment, but ideally it would yes. Once this gets merged the task on the CI will execute `publishRelease` that in turn will scan the changelog file to fill the release and create the tag